src/WebJobs.Script/ManagedDependencies/ManagedDependencyConfigs.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Script.ManagedDependencies
+{
+    internal class ManagedDependencyConfigs : IManagedDependencyConfigs
+    {
+        public string RapidUpdateAzModulePath { get => @"D:\Program Files (x86)\ManagedDependencies\PowerShell\AzPSModules"; }
+
+        public string ManagedDependencyHomeDirectoryWin { get => @"D:\home\manageddependencies"; }
+
+        public string ManagedDependencyHomeDirectoryLinux { get => "/home/manageddependencies"; }
```

Comment:
`D:/Home` and `/home` can be resolved using `Environment.GetEnvironmentVariable(EnvironmentSettingNames.AzureWebsiteHomePath)`

Reply:
for the feedback on using `Environment`, just make sure you're using `IEnvironment`

Reply:
{FunctionApp root}\manageddependencies, the logic is there in the ManagedDependencyProvider:
 //If it's local environment Linux or Windows
                else
                {
                    dependencyDownloadPath = Path.Combine(Path.GetDirectoryName(hostFilePath), ManagedDependenciesPropertiesName);